package.json

```diff
@@ -23,9 +23,10 @@
   },
   "main": "./index.js",
   "bin": {
-    "ghost-inspector": "./bin/cli.js"
+    "ghost-inspector": "./bin/ghost-inspector"
   },
   "dependencies": {
+    "caxa": "^2.1.0",
```

Comment:
Could this potentially be a `devDependency`? Or does it need to be included in the published package?

Reply:
Actually, that doesn't need to be in there at all. We install it manually for packaging.